models/Sketch/Sketch.test.js

```diff
@@ -35,6 +35,14 @@ describe('Sketch', () => {
       });
     });
 
+    it('should work when file is Buffer', () => {
+      expect(true).toBeTruthy();
```

Comment:
This expect is not needed

Reply:
ah- this one was present previously in the test above. I've removed it from that one too.